src/main/java/ome/formats/importer/ImportConfig.java

```diff
@@ -231,7 +231,7 @@ public ImportConfig(final Preferences prefs,
 
         // Various startup requirements
         omeroVersion = ImportConfig.class.getPackage().getImplementationVersion();
-        log.info("OMERO Version: " + omeroVersion);
+        log.info("omero-blitz Version: " + omeroVersion);
```

Comment:
```suggestion
        log.info("OMERO.blitz Version: " + omeroVersion);
```
is an alternative, given https://docs.openmicroscopy.org/omero/5.6/developers/server-blitz.html, but then might get too confusing in distinguishing from OMERO.server?

Reply:
I could see adding omero-model, omero-server and perhaps OMERO.server versions if we can come up with a compact enough representation.